src/app/OperationalDeviceProxy.h

```diff
@@ -268,6 +263,15 @@ class DLL_EXPORT OperationalDeviceProxy : public DeviceProxy,
 
     CHIP_ERROR EstablishConnection();
 
+    /*
+     * This checks to see if an existing CASE session exists to the peer within the SessionManager
```

Comment:
I now understand the name however I believe it makes it sound more complicated than it is. 

s/CheckAndLoadExistingSession/UsePreExistingCASESession/

is likely better.